lib/install/fsmspec.go

```diff
@@ -59,7 +60,16 @@ func FSMSpec(config FSMConfig) fsm.FSMSpecFunc {
 				config.Operator, remote)
 
 		case p.Phase.ID == phases.WaitPhase:
+			client, err := httplib.GetClusterKubeClient(config.DNSConfig.Addr())
```

Comment:
Not sure if @r0mant was going to revert the API change but this now returns 3 values